src/sections/dataset/Dataset.tsx

```diff
@@ -37,7 +38,9 @@ export function Dataset({ datasetRepository, id }: DatasetProps) {
               <Col sm={9}>Citation Block</Col>
             </Row>
             <Row>
-              <Col sm={9}>Summary Block</Col>
```

Comment:
Can we add the `<Col sm={9}>` wrapping the DatasetSummary component to make the page look more like the jsf page?

Reply:
yes, done 👍